kernel/common/enumerable.rb

```diff
@@ -160,6 +160,29 @@ def slice_before(arg = undefined, &block)
     end
   end
 
+  def slice_after(pattern = undefined, &block)
+    pattern_given = !undefined.equal?(pattern)
+    raise ArgumentError, "cannot pass both pattern and block" if pattern_given && block_given?
+    raise ArgumentError, "wrong number of arguments (0 for 1)" if !pattern_given && !block_given?
+    block = Proc.new { |elem| pattern === elem } if pattern_given
```

Comment:
Probably should've been more clear with this (my bad), I meant whitespace between every line, not just the one preceding `Enumerator.new do ...`